newIDE/app/src/UI/Layout.js

```diff
@@ -61,7 +61,7 @@ export const TextFieldWithButtonLayout = ({
   renderButton,
 }: TextFieldWithButtonLayoutProps) => {
   return (
-    <div style={textFieldWithButtonLayoutStyles.container}>
+    <ResponsiveLineStackLayout alignItems="flex-start" expand noMargin>
```

Comment:
yes, justified, this is so that the inlinepopover looks fine/responsive on small screens (see above screnshots)
Nice catch, I'll remove the container